groovy/aws_sso_refresh.groovy

```diff
@@ -112,6 +112,7 @@ node(REFRESH_NODE_LABEL){
                 sh '''
                 #!/bin/bash
                 scp -r ~/.aws yudongsi@mlt-ace:~/
+                scp -r ~/.aws weizhuoz@mlt-ace:~/

```

Comment:
is it for the convenience of your local aws-cli operations?

Reply:
Yes, it is.

Reply:
This change will not work, because the refresh job use remote=false as default.